pam/internal/adapter/brokerselection.go

```diff
@@ -70,6 +72,12 @@ func (m brokerSelectionModel) Init() tea.Cmd {
 func (m brokerSelectionModel) Update(msg tea.Msg) (brokerSelectionModel, tea.Cmd) {
 	switch msg := msg.(type) {
 	case brokersListReceived:
+		if len(msg.brokers) == 0 {
+			return m, sendEvent(pamError{
+				status: pam.ErrAuthinfoUnavail,
+				msg:    "No brokers available",
+			})
+		}
```

Comment:
I'm pretty sure this is unreachable code. Our broker service will always return at least the local broker.

Reply:
This is not hitting the case in our test brokers, but in case of errors it may happen, so could be paranoia but I'd prefer to keep the check.